ftw/file/upgrades/20150504090723_enable_tiny_mce_image_upload/tinymce.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<object>
+ <resourcetypes>
+  <imageobjects purge="True">
+    <element value="File"/>
```

Comment:
@jone @shylux No, imho this is part of the integration. ftw.file `can` act as an image, but this should not be the default behavior.